src/views/_init.js.njk

```diff
@@ -3,6 +3,16 @@ window.addEventListener('load', (event) => {
 
   if (window.GOVUKFrontend) {
     window.GOVUKFrontend.initAll();
+  if(document.querySelector('#current-species-caught')) {
```

Comment:
I'll be pedantic here and suggest `select#current-species-caught` instead of `#current-species-caught` just in case we use the same ID somewhere else. We can get away with being more specific here as we're only interested in enhancing a `select`, so including it in the query should be fine.